komodo/release_transpiler.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+from argparse import ArgumentError, ArgumentParser, ArgumentTypeError
+from komodo import load_yaml, write_to_file, prettified_yaml
+
+rhel7_builtins = {
```

Comment:
This is most likely outdated already. Can you take this information as a file argument, and I will add this file to komodo-releases? So you get

`komodo-transpiler combine --release-base 2020.07.b0 --release-folder releases --override-mapping rhe7-builtins.yml`? What do you think?